static/js/components/EducationForm.js

```diff
@@ -126,8 +129,12 @@ class EducationForm extends ProfileFormFields {
     </Cell>;
   }
 
-  renderEducationLevelEntries(level: any): Array<React$Element<*>|void>|void {
+  renderEducationLevelEntries(level: any): Array<React$Element<*>|void>|void|null {
     const { profile } = this.props;
+    if (!profileOfLoggedinUser(profile) && (!profile.education || profile.education.length === 0)) {
+      return null;
+    }
```

Comment:
This function shouldn't have this control flow in it, it should be a more pure function which just takes a level as an argument and pulls the profile out of `this.props`. See my other comment, but I think this change is confusing. This function should stay the same, and below in the `renderCard` method use the `profileOfLoggedInUser` to check whether it should be called.